frontend/pages/ManageControlsPage/MacOSSettings/MacOSSettings.tsx

```diff
@@ -17,14 +18,20 @@ interface IMacOSSettingsProps {
   // renders this one
   location: {
     query: { team_id?: string };
+    action: string;
   };
 }
 
 const MacOSSettings = ({ params, location }: IMacOSSettingsProps) => {
   const { section } = params;
   const { team_id } = location.query;
+  const { currentTeam } = useContext(AppContext);
```

Comment:
The URL is not being updating by React Router until a moment after the page loads, which causes a flash of "No team" content before changing to the selected team. We need to fix this architecturally, but a stop-gap solution is to double-reference the `currentTeam` in context.